### PR TITLE
raidboss: add basic p3s timeline/triggers

### PR DESCRIPTION
**Pull request by quisquous**

### CODE REVIEW

**Comment by quisquous** [comment]
Triggers need some testing.  Timeline made it through a lot of logs, so feel pretty confident on that.

**Comment by tridisaster** [comment]
how do i add this to cactbot? i made the files for p3s and placed them in cactbot/ui/raidboss/data/06-ew/raid   but they wont show up on cactbot under "raidboss"

**Comment by Legends0** [comment]
@tridisaster you need to install NodeJS, pull latest from git, npm install, then npm start. Once npm finishes compiling, it will host a web service on your localhost that you then reconfigure cactbot to look at, such as http://localhost:8080/ui/raidboss/raidboss.html instead of the raidboss.html from the release. The release ball has a raidboss.html that loads a precompiled raidboss_data.bundle.js.

**Comment by Legends0** [comment]
> Triggers need some testing. Timeline made it through a lot of logs, so feel pretty confident on that.

I will be able to test this out tomorrow.

File: ui/raidboss/data/06-ew/raid/p3s.txt [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+828.7 "Flare of Condemnation/Sparks of Condemnation" sync / 1[56]:[^:]*:Phoinix:(66FE|66FF):/
+831.8 "--targetable--"
+839.9 "Devouring Brand" sync / 1[56]:[^:]*:Phoinix:66CE:/
+847.0 "Experimental Fireplume" sync / 1[56]:[^:]*:Phoinix:66BE:/
+856.9 "Devouring Brand" sync / 1[56]:[^:]*:Phoinix:6D83:/
+857.2 "Searing Breeze" sync / 1[56]:[^:]*:Phoinix:6704:/
+865.4 "Left Cinderwing/Right Cinderwing" sync / 1[56]:[^:]*:Phoinix:(6702|6703):/
+876.5 "Scorched Exaltation" sync / 1[56]:[^:]*:Phoinix:6706:/
+883.6 "Scorched Exaltation" sync / 1[56]:[^:]*:Phoinix:6706:/
+# TODO: enrage
```
[the remark this answers is not in the document]
Comment (quisquous) [reply ui/raidboss/data/06-ew/raid/p3s.txt]: Thank you, that's perfect.

**Comment by quisquous** [comment]
> > Triggers need some testing. Timeline made it through a lot of logs, so feel pretty confident on that.
> 
> I will be able to test this out tomorrow.

I gave this a spin tonight.  Everything was fine, other what I changed in 5f36bdd.  I gave it a test through the raid emulator, and everything seemed reasonable to me.

**Pull request merged**